Add 11.34 release notes (#498)
mondoohq/docs: releases/2024-12-10-mondoo-11.34-is-out.md

---
slug: mondoo-11.34-is-out/
title: Mondoo 11.34 is out!
description: Announcing the 11.34 release of Mondoo with updated CIS macOS Sequoia benchmarks, and more!
authors: [tim]
image: /img/featured_img/mondoo-feature.jpg
tags: [release, mondoo]
---

> ## 🥳 Mondoo 11.34 is out! This release includes updated CIS macOS Sequoia benchmarks, and more!

Reviewer: ```suggestion
## 🥳 Mondoo 11.34 is out! This release includes updated CIS macOS Sequoia benchmarks and more!
```

Get this release: [Installation Docs](https://mondoo.com/docs/cnspec/) | [Package Downloads](https://releases.mondoo.com/cnspec/) | [Docker Container](https://hub.docker.com/r/mondoo/cnspec)

---

## 🎉 NEW FEATURES

## 🧹 IMPROVEMENTS

### CIS macOS Sequoia benchmark 1.0

> Secure Apple's latest macOS release with an updated CIS benchmark including several new recommendations and improved queries compared to the preview release.

Reviewer: ```suggestion
Secure Apple's latest macOS release with an updated CIS benchmark. This policy has several new recommendations and improved queries compared to the preview release.
```

## 🐛 BUG FIXES AND UPDATES

- Improve display of affected assets in the vulnerabilities overview page.
- Improve default values displayed for Cloudflare R2 buckets.
- Handle nil login values in the `microsoft.user.auditlog` resource.
- Add more detailed command line help for the GitLab provider.
- Fix failures with JSON exports if a duplicate key exists.
- Fix failures in the Mondoo Linux Security Policy's auditd checks when scanning Ubuntu or Debian systems.

> - Fix failures CIS VMware policy vulnerabilities checks.

Reviewer: ```suggestion
- Fix failures running CIS VMware policy vulnerabilities checks.
```
